test/unit/callbacks_and_delays_test.lua

```diff
@@ -107,3 +108,43 @@ function g.test_delays_and_scan_callbacks(cg)
     t.assert_equals(full_scan_success_counter, 2)
     t.assert_equals(space:count{}, 0)
```

Comment:
Code style [2]. Let's fix that in this commit too.
```suggestion
    t.assert_equals(space:count(), 0)
```